api/sixtyfps-rs/migration.md

```diff
@@ -12,13 +12,15 @@ In 0.2.0 we have increased the minimum version of rust. You need to have rust co
 
 #### Models
 
+##### `Model::row_data`
```

Comment:
That's quite some nesting.

I think you can remove the ### Rust API nesting

Reply:
That would make sense, yes:-)

I'll do that after merging this, I do not want to  trigger another CI run of this patch.